aten/src/ATen/native/nested/NestedTensorMath.cpp

```diff
@@ -1376,5 +1380,109 @@ Tensor reshape_as_nested(const Tensor& self, const Tensor& other) {
   return self.reshape(sizes);
 }
 
+TensorOptions verify_empty_parameters(
```

Comment:
can you move this work to NestedTensorUtils.cpp

If you wanted to be a really good steward you could create a new file could NestedTensorFactories.cpp and NestedTensorFactories.h but not needed for this PR